src/main/resources/fixture-build-map.json

```diff
@@ -0,0 +1,849 @@
+{
```

Comment:
Looks like this is a list of all the old system's build numbers + commit hashes yeah?

Reply:
yeah up to 1801 for now